Core/Config.cpp

```diff
@@ -443,9 +435,7 @@ static ConfigSetting graphicsSettings[] = {
 #endif
 
 	// TODO: Replace these settings with a list of options
-	ConfigSetting("PartialStretch", &g_Config.bPartialStretch, &DefaultPartialStretch, true, true),
-	ConfigSetting("StretchToDisplay", &g_Config.bStretchToDisplay, false, true, true),
-	ConfigSetting("SmallDisplayZoom", &g_Config.iSmallDisplayZoom, 0, true, true),
+	ConfigSetting("SmallDisplayZoom", &g_Config.iSmallDisplayZoom, 2, true, true),
```

Comment:
I might have a weird concept of order and simplicity since to me all of those settings being used in one function affecting same thing felt natural, but I don't mind dividing it. I guess types would be same as now plus manual scaling and show zoom level options separately only when manual scaling is selected.